cherryontech/do-five-things
# chore: Display draft vue components

## Context
Devs trying to test RoR + Vue setup 😉 

## Work done
- Added drafts of some basic SFCs and utils
- Demo draft of the components on the index page
- Added some basic global styles

## Manual testing instructions
1. run `./bin/env` to serve the app
2. Navigate to the index page of the app
- Expect to see some interactive component examples
